### DISCUSSION TITLE
Default hyperlink color in dark mode isn't as accessible as it could be

### DISCUSSION THREAD

**Discussion by barbaricyawps** [pradyunsg/furo#511] [Potential Issue]
Hello! We love the Furo theme over at the Salt Project. Thanks for all your hard work on it.

I wanted to start a discussion/open an issue to talk about accessibility concerns for the default hyperlink color in dark mode. We received some feedback from community members that the dark mode links were hard to read and we implemented an override to fix that. I'd recommend possibly making a change upstream at the theme level too.

If I'm reading the CSS correctly, here's the current hyperlink colors (bg color #131416, link color #368CE2):
https://webaim.org/resources/contrastchecker/?fcolor=368CE2&bcolor=131416

The default dark mode theme has a WACG contrast score of about 5.27:1. That's a passing score, but has room for improvement.

Here's the hyperlink colors that we implemented (bg color #131416, link color #66CCF4):
https://webaim.org/resources/contrastchecker/?fcolor=66CCF4&bcolor=131416

Our update dark mode theme has a WACG contrast score of 10.1:1, which is much higher. I tested it with our community members who initially complained about the hyperlink colors and they said it was much better. You might want to consider implementing a similar change for the defaults here?

--------

I have one more related issue to mention. In the course of implementing our theme overrides, I got in a debate with some of my team members about whether visited site links should change their color or not. We escalated the debate to our UX team, and they recommended using a different color for visited links. They cited this Nielsen Norman group article about it: https://www.nngroup.com/articles/change-the-color-of-visited-links/

Again, I'm not 100% sure if the Furo theme gives the ability to set a separate color variable for visited links. That could be another feature to consider.

**Comment by pradyunsg** [comment]
> 10.1:1

While going up to 10 is... a lot. I've gone ahead and tweaked the colors in ae2107702a26a9556f609fc818dc43e4f5a7f315 along with adding the visited colors on links.

Thanks for providing all the context here so that I could action on this as soon as I read this! ❤️

**Reply by barbaricyawps** [reply]
Awesome, @pradyunsg . Thanks so much!
